tasks/install_all_vms.yml

```diff
@@ -0,0 +1,6 @@
+- name: Push VM configuration for {{ vm.host }} @{{ vm.org }}
```

Comment:
Why is this even an own file?

Reply:
What you mean?

Reply:
Why do you have a file for just two tasks?